pkg/receive/handler.go

```diff
@@ -275,6 +277,18 @@ func NewHandler(logger log.Logger, o *Options) *Handler {
 		),
 	)
 
+	h.router.Post(
```

Comment:
@saswatamcode just FYI use this endpoint naming will require users to use the `metrics_endpoint` instead of `endpoint` property, maybe worth document it or writing a blog post later about Thanos and OTLP wdyt?